### PR TITLE
Set --playbook-dir in calls to ansible-inventory

### PR DESCRIPTION
**Pull request by AlanCoding**
##### SUMMARY
Connect https://github.com/ansible/awx/issues/3113

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - API

##### AWX VERSION
```
3.0.1
```

##### ADDITIONAL INFORMATION
For that use case to work, users will still have to wait on the fix from Ansible core. However, nothing is blocking us from implementing `--playbook-dir`, which can be done with or without that fix.

Also, the `--playbook-dir` option does not exist in Ansible 2.4. So that makes this implementation much, much, more complicated.

### CODE REVIEW

File: awx/main/utils/common.py
Comment (ryanpetrello) [inline awx/main/utils/common.py]: Just do it already @AlanCoding 😂

**Review by ryanpetrello — APPROVED** [review]

**Comment by AlanCoding** [comment]
I'm only doing one last bit of testing, and then I'll be ready to merge this.

**Comment by AlanCoding** [comment]
Just finished testing, looks like this is ready to go.

File: awx/main/management/commands/inventory_import.py
Comment (AlanCoding) [inline awx/main/management/commands/inventory_import.py:138]: ugh, this is still technically blocked by

https://github.com/ansible/ansible/pull/51177

Yes, the flag existed in 2.5, but it didn't necessary work right. I don't know what use case we need that for.

We do kind of need to ask if they intend to backport that.

I'm of a mind to only set this if the version is 2.8 or over, and do this along with verbosity.

Comment (AlanCoding) [reply awx/main/management/commands/inventory_import.py:138]: The question has been asked in the Ansible link.

**Comment by AlanCoding** [comment]
I'm happy with this, but @chrismeyersfsu would _not_ be happy with me if I merged it.

Putting on blocked label until runner integration is merged.

**Comment by AlanCoding** [comment]
this should still delay until inventory plugins merge

**Pull request merged**
